ncm-symlink/src/main/perl/symlink.pod

```diff
@@ -17,9 +17,9 @@ symlink : symlink NCM component.
 
 Object to create/delete symbolic links. When creating symlinks, target existence can be checked. And clobbering can be disabled.
 
-Target definition may contain contextual variable, using syntax '{variable}'. Variables are defined globaly for all links and allow to use the same target definition for different contexts (e.g. platforms).
+Target definition may contain contextual variable, using syntax C<{variable}>. Variables are defined globaly for all links and allow to use the same target definition for different contexts (e.g. platforms).
 
-Target definition and contextual variable value may also contain expression interpreted as shell commands (.eg. uname). These expressions must enclosed between pair of '@@'. Unless shell command must reevaluated for each link, it is better to associate the shell command with a contextual variable and use the variable in the target definition, as a contextual variable is evaluated once (global).
+Target definition and contextual variable value may also contain expression interpreted as shell commands (.eg. uname). These expressions must enclosed between pair of C<@@>. Unless shell command must reevaluated for each link, it is better to associate the shell command with a contextual variable and use the variable in the target definition, as a contextual variable is evaluated once (global).
```

Comment:
`contain expression` -> either `an expression` or `expressions`
`.eg.` -> `e.g.`
an example of `enclosed between pair of C<@@>` with uname?
`must` -> `must be`

when is the contextual variable evaluated? at the begin or as encountered during the run?

(and maybe some newlines, this is a very long line)